### ISSUE TITLE
How to set the parameters of w8a8 to improve quantization performance

### ISSUE THREAD

**Issue by buxianggaimingzi** [quic/aimet#2439]
The situation I was faced with was that the model got good results with w8a16 quantization accuracy, but when I tried to use w8a8 quantization, compared with w8a16 quantization, the model, data, code and parameters were all exactly the same, only activate_bitwise changed from 16 to 8, but the result was very poor. What I want to know is: What does w8a8 quantization need to do to approximate the quantization result of w8a16?

and here is my quant code, and the config file directly uses the default config parameter
``` python
#fp_model = my_ori_model
#ptq_dataloader = ...
#dummy_shape = ...
#dummy_input = torch.randn(dummy_shape)

equalize_model(fp_model, .dummy_shape)
quant_params = QuantParams(weight_bw=8, act_bw=8, config_file=config_path)
correct_bias(model=fp_model, quant_params=quant_params, num_quant_samples=1000,
                     data_loader=ptq_dataloader, num_bias_correct_samples=1000)

quant_sim = QuantizationSimModel(model=fp_model,
                                             config_file=config_path,
                                             dummy_input=dummy_input,
                                             rounding_mode='nearest',
                                             default_output_bw=8,
                                             default_param_bw=8,
                                             in_place=False,
                                             quant_scheme=QuantScheme.training_range_learning_with_tf_init)
quant_sim.compute_encodings(forward_pass_callback=validate, forward_pass_callback_args=12)

```

Then the training of qat is initialized with the ptq model

``` python
#qat_dataloader = ...
#qat_epoch = ...
#fx_optimizer = ...
#fx_scheduler = ...

fx_model = quant_sim.model
fx_model.train()

for epoch in range(1, qat_epoch +1):
    for idx, train_data in enumerate(qat_dataloader):
        fx_optimizer.zero_grad()

        gt = train_data[0]
        input = train_data[1]

         fx_output = fx_model(input)
         with torch.no_grad():
                fp_output = fp_model(input)
         fx_fp_loss = F.l1_loss(fp_output, fx_output, reduction='mean')

        loss.backward()
        fx_optimizer.step()
        fx_scheduler.step()
```

**Comment by quic-mangal** [comment]
Hi @buxianggaimingzi, if you quantize to a lower BW, then you will get degraded results. If you are ok to use mixed precision in your model, you could try to put a few layers in W8A16 and see if performance improves. You could also try out Adaround technique to improve the performance.

**Comment by buxianggaimingzi** [comment]
Thanks for your reply, maybe this model is not suitable for the quantization of w8a8 accuracy, I will try the qat of mixed accuracy, thank you

**Comment by quic-hitameht** [comment]
Closing this as resolved. If there is another question please re-open or create a new issue.
